src/settingsholder.cpp

```diff
@@ -133,6 +139,24 @@ void SettingsHolder::clear() {
   // We do not remove language, ipv6 and localnetwork settings.
 }
 
+// Returns a Report which settings are set
+// Used to Print in LogFiles:
+QString SettingsHolder::getReport() {
+  QString buff = "\n\nSettings \n======== \n";
```

Comment:
Instead of appending strings, you can do:

```
QString buffer;
QTextStream out(&buffer);
buff << setting << "-> <Sensitive>" << Qt::endl;
...
```